stories/features/rewards/concepts.tsx

```diff
@@ -83,6 +83,9 @@ storiesOf('Feature Components/Rewards/Concepts/Desktop', module)
   .add('Site Banner', withState({ donationAmounts, currentAmount: '5.0', showBanner: true }, (store) => {
     const screenName = text('Screen Name', '')
     const tweetText = text('Tweet Text', '')
+    const redditUserName = text('Reddit User Name', '')
```

Comment:
Let's try to use same knobs and have select with types

Reply:
Done